### PR TITLE
1835: Small corrections to private powers in regard to share ownership

### PR DESCRIPTION
**Pull request by neutronc**
Private Bavarian Companies come with 10 percent  of the Bavarian Corporation

### CODE REVIEW

File: lib/engine/config/game/g_1835.rb [hunk 1]
```diff
@@ -197,15 +197,24 @@ module G1835
           ],
           "free": true,
           "count": 1
-        }
+        },
```
Comment (perwestling) [inline lib/engine/config/game/g_1835.rb:200]: Formatting seems a bit off, here and below.

**Review by perwestling — APPROVED** [review]
This seems to be on the smaller side for PRs.... There is some overhead with review, merge, deploy so I think it is a good idea to not have too many PRs. At the same to it is good to break down into managable PRs.

File: lib/engine/config/game/g_1835.rb (hunk 1 above)
Comment (jenf) [reply lib/engine/config/game/g_1835.rb:200]: Please fix the probably tabs/space issue, then it's good to go.

**Comment by neutronc** [comment]
Cant get the merge out of this one, sorry. Next time i'll remember to rebase my fork :(

**Review by tobymao — APPROVED** [review]

**Pull request merged**